specification/specs/query_dsl/compound/dismax/DisMaxQuery.ts

```diff
@@ -1,4 +1,5 @@
 class DisMaxQuery {
 	queries: QueryContainer[];
 	tie_breaker: double;
+  boost: float;
```

Comment:
`tab` (do we have `.editorconfig` configured correctly?)

Reply:
We do have. it, but the input spec is in the wrong format.
I'll open a separate pr to address this.